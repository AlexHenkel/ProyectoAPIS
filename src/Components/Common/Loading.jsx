import React from 'react'
import styled from 'styled-components'
import { LinearProgress } from 'material-ui/Progress'

const Container = styled.div`
  width: 100%;
  margin-top: 30px;
`

export default () => (
  <Container>
    <LinearProgress color="accent" />
  </Container>
)
