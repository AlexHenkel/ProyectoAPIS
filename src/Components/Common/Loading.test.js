import React from 'react'
import Loading from './Loading'

describe('Loading component', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<Loading />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
