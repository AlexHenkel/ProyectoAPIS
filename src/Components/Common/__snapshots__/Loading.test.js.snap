// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loading component renders correctly 1`] = `
<div
  className="sc-bdVaJa hjDGfY"
>
  <div
    className="MuiLinearProgress-root-1 MuiLinearProgress-accentColor-5"
  >
    <div
      className="MuiLinearProgress-bar-8 MuiLinearProgress-accentColorBar-6 MuiLinearProgress-indeterminateBar1-13"
      style={Object {}}
    />
    <div
      className="MuiLinearProgress-bar-8 MuiLinearProgress-accentColorBar-6 MuiLinearProgress-indeterminateBar2-14"
      style={Object {}}
    />
  </div>
</div>
`;
